packages/mongo-nav/src/org-nav/OrgNav.tsx

```diff
@@ -227,6 +226,14 @@ function OrgNav({
     );
   }
 
+  let displayProjectAccess: boolean;
+
+  if (onPremEnabled) {
+    displayProjectAccess = currentProjectName !== 'None' ? true : false;
+  } else {
+    displayProjectAccess = showProjectNav;
+  }
```

Comment:
Can define the happy path as the default path.

```suggestion
  let displayProjectAccess = showProjectNav;

  if (onPremEnabled) {
    displayProjectAccess = currentProjectName !== 'None' ? true : false;
  }
```

Reply:
Why is it being compared against the string 'None'? Someone could name their project that. We should check if the currentProjectId is set no?

Reply:
Checking against none currently bc that's the value we're setting as default. You're right, smarter to check against id. Will update now.

Reply:
Updated!